gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.jvmargs=-Xmx2g -XX:MaxPermSize=512m
```

Comment:
Looks like there are even some tests failing now, so we need to address this problem somehow.